protocol/v2/ssv/spectest/ssv_mapping_test.go

```diff
@@ -96,18 +95,6 @@ func prepareTest(t *testing.T, logger *zap.Logger, name string, test interface{}
 			typedTest.Tests = append(typedTest.Tests, msgProcessingSpecTestFromMap(t, subtest.(map[string]interface{})))
 		}
 
-		return &runnable{
-			name: typedTest.TestName(),
-			test: func(t *testing.T) {
-				typedTest.Run(t)
-			},
-		}
-	case reflect.TypeOf(&partialsigmessage.MsgSpecTest{}).String(): // no use of internal structs so can run as spec test runs
```

Comment:
@MatusKysel FYI: we don't have any cases of `partialsigmessage.MsgSpecTest`type tests

Reply:
can you add this change to the description..

Reply:
Added